test/opt/ccp_test.cpp

```diff
@@ -925,6 +925,43 @@ TEST_F(CCPTest, FoldWithDecoration) {
   SinglePassRunAndMatch<CCPPass>(text, true);
 }
 
+// Reduced from https://github.com/KhronosGroup/SPIRV-Tools/issues/3343.
+// Comparisons using 64-bit types should not be foldable even if their operands
+// are constant.
+TEST_F(CCPTest, Reject64BitComparisons) {
```

Comment:
I don't think the test should be here.  We should be testing the instruction folder to make sure it does the right thing.  This is a bug in the instruction folder that was fixed in code that is called from the instruction folder.  Add the test to the `IntegerArithmeticTestCases` in the fold_test.cpp.

Reply:
Thanks.  Moved.